dev-python/spyder-autopep8/spyder-autopep8-9999.ebuild

```diff
@@ -0,0 +1,20 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_6 )
+
+inherit distutils-r1 git-r3
+
+DESCRIPTION="Plugin to run the autopep8 python linter from within the spyder editor"
+HOMEPAGE="https://github.com/spyder-ide/spyder-autopep8"
+EGIT_REPO_URI="https://github.com/spyder-ide/spyder-autopep8.git"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS=
+
+RDEPEND="
+	dev-python/autopep8[${PYTHON_USEDEP}]
+	<dev-python/spyder-4.0.0[${PYTHON_USEDEP}]"
```

Comment:
... although I do wonder if it's worth it to add these at all? Ideally you'd add spyder-4.0.1, then stabilize it after ~30 days and drop every older version.